### PR TITLE
feat: add browser field

### PR DESCRIPTION
**Pull request by poppinlp**
Hi.
This PR just add a `browser` field and little update for test case.
Sorry for not open an issue before PR. May you like it. :)

### CODE REVIEW

**Comment by poppinlp** [comment]
The second commit includes:

- update dependencies to run tests on node 6
- add node 8 in travis test since it's still alive

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 language: node_js
 node_js:
-  - "node"
+  - "stable"
```
Comment (typicode) [inline .travis.yml:3]: Used to put `"stable"` but changed to `"node"` as it's what they use in https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions (not a blocker though :))

**Comment by typicode** [comment]
Thanks a lot 👍 I'll merge later

**Comment by typicode** [comment]
And thanks also for the dependencies upgrade.

**Pull request merged**